src/main/java/com/kryptokrauts/aeternity/sdk/domain/ObjectResultWrapper.java

```diff
@@ -5,10 +5,8 @@
 import lombok.experimental.SuperBuilder;
 
 /**
- * this object result wrapper introduces the error handling when returning simple string from node
+ * This object result wrapper introduces the error handling when returning simple string from node
```

Comment:
do we return a string here? is this correct?

Reply:
copy paste from the StringResultWrapper ;)